docs/installation/server-config/ccloud-ksql.rst

```diff
@@ -0,0 +1,44 @@
+.. _install_ksql-ccloud:
+
+Using KSQL with |ccloud|
+========================
+
+You can connect KSQL to your |cp| Kafka cluster in |ccloud|.
```

Comment:
Is that the phrasing we use to describe CCloud endpoints?

Reply:
Also, I would explicitly mention in this document that only the configuration of KSQL server(s) must be configured for CCloud usage.  (The KSQL CLI does not need to be aware of CCloud.)

Reply:
@apurvam wydt `Is that the phrasing we use to describe CCloud endpoints?` ^

Reply:
Once this is merged we should update the [faq.rst ](https://github.com/confluentinc/ksql/blob/4.1.x/docs/faq.rst#id33) to reference this section and remove the actual config from the FAQ.

Reply:
We can add a sentence on line 12 saying " Note that only the ksql server needs to be specially configured to work with ccloud. In particular, the ksql cli does not need any special configuration".

Reply:
@apurvam done https://github.com/confluentinc/ksql/pull/955/commits/70e719cf685cec84a082874d0dd7fa5910e56005